### PR TITLE
rpolars_style 0.1.1: fix styler `<<-`   and style all files 

### PR DESCRIPTION
**Pull request by sorhawell**
update to rpolars_style  to not convert  the `<<-`  operator to `=` in e.g. `utils.R`

### CODE REVIEW

**Comment by sorhawell** [comment]
follow up to #594

**Comment by sorhawell** [comment]
styled all files with this

```R
source("dev/styler_utils.R")
style_files(list.files(".",full.names=TRUE, pattern = "\\.R$|\\.Rmd$", recursive = TRUE))
```

File: R/extendr-wrappers.R
Comment (eitsupi) [inline R/extendr-wrappers.R:1]: This file should not be updated.

**Review by eitsupi — CHANGES_REQUESTED** [review]

File: R/extendr-wrappers.R
Comment (sorhawell) [reply R/extendr-wrappers.R:1]: reverted file, and added function `style_entire_pkg()`

```R
style_entire_pkg = function() {
  list.files(".", full.names = TRUE, pattern = "\\.R$|\\.Rmd$", recursive = TRUE) |>
    setdiff("./R/extendr-wrappers.R") |>
    style_files()
}
```

**Review by eitsupi — APPROVED** [review]
Thanks, let's try merging.

**Comment by sorhawell** [comment]
for reference added function to style all files 
```R
source("dev/styler_utils.R")
style_entire_pkg()
```

instead of just styling files of a given PR
```R
style_files()
```

**Pull request merged**